android-sdk/Dockerfile

```diff
@@ -78,5 +78,9 @@ ENV LANG en_US.UTF-8
 # install Firebase CLI
 RUN npm install -g firebase-tools
 
+# install Google Cloud SDK
+RUN apk add --update \ python \ curl \ which \ bash
```

Comment:
Why the `\`?

```suggestion
RUN apk add --update python curl which bash
```

Reply:
In addition, `apk` is for Docker alpine, this might not work as the image is `ubuntu:18.04`.

I would move this packages to L19-L27